docs/APP.md

```diff
@@ -46,16 +46,28 @@ The app communicates with the wrapper using a messaging provider. The default pr
 
 To send an intent to the wrapper (i.e. invoke a method on your smart contract), simply call it on the instance of this class as if it was a JavaScript function.
 
-For example, to invoke `increment` on your app's smart contract:
+For example, to execute the `increment` function in your app's smart contract:
 
 ```js
 const app = new AragonApp()
 
 // Sends an intent to the wrapper that we wish to invoke `increment` on our
 // app's smart contract
-app.increment()
+app.increment(1)
 ```
 
+You can also pass an optional object after all the required function arguments specify some values that will be sent in the transaction. They are the same values that can be passed to `web3.eth.sendTransaction()` and can be checked in this [web3.js document](https://web3js.readthedocs.io/en/1.0/web3-eth.html#id62).
+
+```js
+app.increment(1, { gas: 200000, gasPrice: 80000000 }) // careful: hardcoding the gas limit could break your app!
+```
+
+Some caveats to customizing transaction parameters:
+
+- `from`, `to`, `data`: will be ignored as aragon.js will calculate those.
+- `gas`: The gas amount required for an action will vary depending on how the action ends up being executed (if it goes through a forwarder it will require more gas). Manually specifying the gas limit is not recommended as it could break your app depending on what DAOs it gets installed to.
```

Comment:
In the future, it may be interesting to add something like `minGas` to specify a lower bound on gas. I think the cases where gas estimation get stuck is when it estimates a really low amount of gas so this could avoid that while still allowing for higher cost operations like forwarding.

Reply:
Great idea. Created an issue so we don't forget about it https://github.com/aragon/aragon.js/issues/150. Also thinking about whether we should always